packages/maskbook/src/_locales/en/messages.json

```diff
@@ -315,10 +315,19 @@
     "plugin_wallet_on_connect": "Connect Wallet",
     "plugin_wallet_wrong_network": "Wrong Network",
     "plugin_wallet_setup_title_create": "Create a Wallet",
+    "plugin_wallet_create_import_choose": "Create Wallet & Import Wallet",
+    "plugin_wallet_create_a_new_wallet": "Create a New Wallet",
+    "plugin_wallet_create_a_new_wallet_description": "Mask network supports Eth and Polka networks ",
```

Comment:
Should it use another slogan? We don't support the Polka network.

Reply:
Any suggestion?

/cc @xiazhiming

Reply:
Text got updated, and `Polka` network was removed.